server/parsec/asgi/administration.py

```diff
@@ -561,11 +575,183 @@ async def administration_organization_sequester_services(
         case SequesterGetOrganizationServicesBadOutcome.ORGANIZATION_NOT_FOUND:
             raise HTTPException(status_code=404, detail="Organization not found")
         case SequesterGetOrganizationServicesBadOutcome.SEQUESTER_DISABLED:
-            cooked_services = []
+            raise HTTPException(status_code=400, detail="Sequester disabled")
 
     return JSONResponse(
         status_code=200,
         content={
             "services": cooked_services,
         },
     )
+
+
+class SequesterServiceConfigInStorage(BaseModel):
+    model_config = ConfigDict(strict=True)
+    type: Literal["storage"]
+
+    @property
+    def cooked(self) -> SequesterServiceConfig:
+        return SequesterServiceType.STORAGE
+
+
+class SequesterServiceConfigInWebhook(BaseModel):
+    model_config = ConfigDict(strict=True)
+    type: Literal["webhook"]
+    webhook_url: str
+
+    @property
+    def cooked(self) -> SequesterServiceConfig:
+        return (SequesterServiceType.WEBHOOK, self.webhook_url)
+
+
+SequesterServiceConfigField = Annotated[
+    Union[SequesterServiceConfigInStorage, SequesterServiceConfigInWebhook],
+    Field(
+        discriminator="type",
+    ),
+]
+
+
+class SequesterServiceCreateIn(BaseModel):
+    model_config = ConfigDict(strict=True)
+    service_certificate: Base64Bytes
+    config: SequesterServiceConfigField
+
+
+@administration_router.post(
+    "/administration/organizations/{raw_organization_id}/sequester/services"
+)
+@log_request
+async def administration_organization_sequester_service_create(
+    raw_organization_id: str,
+    body: SequesterServiceCreateIn,
+    auth: Annotated[None, Depends(check_administration_auth)],
+    request: Request,
+) -> Response:
+    backend: Backend = request.app.state.backend
+
+    organization_id = parse_organization_id_or_die(raw_organization_id)
+
+    outcome = await backend.sequester.create_service(
+        now=DateTime.now(),
+        organization_id=organization_id,
+        service_certificate=body.service_certificate,
+        config=body.config.cooked,
+    )
+    match outcome:
+        case SequesterServiceCertificate():
+            pass
+        case SequesterCreateServiceValidateBadOutcome.INVALID_CERTIFICATE:
+            raise HTTPException(status_code=400, detail="Invalid certificate")
+        case SequesterCreateServiceStoreBadOutcome.ORGANIZATION_NOT_FOUND:
+            raise HTTPException(status_code=404, detail="Organization not found")
+        case SequesterCreateServiceStoreBadOutcome.SEQUESTER_DISABLED:
+            raise HTTPException(status_code=400, detail="Sequester disabled")
```

Comment:
If this error means the organization does not have a sequester service, why would the "create" route return a "sequester disabled" response?

Reply:
This error is about the fact the organization is not a sequestered one (i.e. it doesn't have a sequester authority configured).